performance-benchmarks/README.md

```diff
@@ -0,0 +1,9 @@
+# Ballerina-performance-benchmarks
+
+To run the performance benchmarks:
+
+mvn clean install -P benchmark -Dwarmup.iterations=<No_of_warmUp_Iterations> -Dbenchmark.iterations=<No_of_benchmark_Iterations>
```

Comment:
Use markdown styles for code: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/7007/commits/61b4cb620114cbedcf70d26dd22248430702a405